### PR TITLE
pdfposter: add page

### PR DESCRIPTION
**Pull request by sbrl**
Just found this useful command - simple, but effective. Not sure what it means by 'art box' in the help.

----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

File: pages/common/pdfposter.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# pdfposter
+
+> Convert a large-sheeted pdf into multiple A4 tiles for printing.
```
Comment (agnivade) [inline pages/common/pdfposter.md]: Should we change it to A4 pages instead of tiles?

Comment (sbrl) [reply pages/common/pdfposter.md]: yeah, that's a good idea

**Comment by sbrl** [comment]
@agnivade In the first instance, An A2 sheet is split into multiple A4 pages for printing - i.e. it's split up into 4 A4 pages.

In the second instance, An A4 page is scaled up to A3, and _then_ split into multiple A4 pages for printing - i.e. A4 -> scaled to A3 -> split into 2 A4 pages.

**Comment by agnivade** [comment]
I meant where are the nos. 4 and 2 coming from ? In the first command, you don't specify 4 anywhere.

For the next one, I understand that you are scaling up to twice the size. And then printing the 2 pages.

**Comment by sbrl** [comment]
@agnivade Ahhh, I see your question now. The numbers 4 and 2 come from the number of A4 sheets the respective paper sizes equal - A3 = 2 x A4,  A2 = 4 x A4, etc. - as `pdfposter` does not appear to have an option to configure this. It simply splits a large page into the number of respective A4 pages required to represent it for printing - after applying the scale factor.

**Comment by sbrl** [comment]
Merging because this PR has been sitting around for ages :P

**Pull request merged**